sdk/nodejs/project.json

```diff
@@ -25,6 +25,12 @@
             }
         },
         "test": {
+            "executor": "nx:run-commands",
+            "options": {
+                "command": "NODE_OPTIONS=--experimental-vm-modules yarn nx test-jest nodejs"
```

Comment:
this is just an environment variable, there should be a way to configure the jest task's environment variables using Nx itself

Reply:
I don't see a way to do that. Will take another look.

Reply:
https://nx.dev/recipes/tips-n-tricks/define-environment-variables